### DISCUSSION TITLE
Unable to recognize podman events from registrator

### DISCUSSION THREAD

**Discussion by rejoycherian** [containers/podman#19857] [Q&A]
Hello, 

I am using gliderlabs\registrator , consul setup for discovery of containers. I am running podman as a  service. So /var/run/podman/podman.sock is made available to registrator (/tmp/docker.sock). I am using Podman API to launch containers from remote. 

The podman  containers are working fine. But the new one's aren't picked by registrator. Can anyone help me in understanding what is the root cause for this?

Note: While restarting the registrator the new containers are picked up and registered.

**Comment by rhatdan** [comment]
Could Registrator be looking for events and not seeing them?
